test/emulator_backend.go

```diff
@@ -233,14 +343,55 @@ func NewEmulatorBackend(
 		logCollection:   logCollectionHook,
 		clock:           clock,
 		contracts:       map[string]common.Address{},
-		accounts:        map[common.Address]*stdlib.Account{},
+		keys:            map[common.Address]*stdlib.PublicKey{},
 		locationHandler: locationHandler,
+		chain:           selectedChain,
+		forkEnabled:     forkEnabled,
+	}
+
+	// Bootstrap accounts only in non-fork mode, for fork mode transaction signatures are not validated & any account can be used.
+	if !forkEnabled {
```

Comment:
Maybe combine this with the `if forkEnabled` above, i.e. put it in the else branch